packages/experimental/Avatar/src/Avatar.tsx

```diff
@@ -0,0 +1,162 @@
+/** @jsx withSlots */
+import { compose, UseSlots, buildProps, mergeProps, withSlots } from '@fluentui-react-native/framework';
+import { Image, NativeModules, ViewProps } from 'react-native';
+import * as React from 'react';
+import { ensureNativeComponent } from '@fluentui-react-native/component-cache';
+
+const avatarName = 'MSFAvatarView';
+
+const NativeAvatarView = ensureNativeComponent(avatarName);
+
+export type Size = 'xSmall' | 'small' | 'medium' | 'large' | 'xLarge' | 'xxLarge';
+
+export type AvatarStyle = 'circle' | 'square';
+
+export type Presence = 'none' | 'available' | 'away' | 'doNotDisturb' | 'outOfOffice' | 'offline' | 'unknown' | 'unknown' | 'blocked';
+
+interface ExportedConstants {
+  sizes: { [key in Size]: number };
+}
+
+const ExportedNativeConstants: ExportedConstants = NativeModules.MSFAvatarViewManager;
+
+export type AvatarData = {
+  /**
+   * The primary text to create initials with (e.g. a name)
+   */
+  primaryText?: string;
+
+  /**
+   * The secondary text to create initials with if primary text is not provided (e.g. an email address)
+   */
+  secondaryText?: string;
+
+  /**
+   * The image to be displayed
+   */
+  image?: Image;
```

Comment:
We need to make sure we can set a base64 encoded image here, as per requirements.

Reply:
If we make this an `ImageURISource`, then we can easily do that by specifying a [data:... URI](https://en.wikipedia.org/wiki/Data_URI_scheme).